src/Header/Header.tsx

```diff
@@ -1,21 +0,0 @@
-import GameLink from './GameLink';
```

Comment:
This `Header` was split up into `Menu` and `Home`.
`Menu` is meant to be used as a generic menu component, allowing each page to define its own menu items.
`Home` is one such implementation, defining the menu (and some text) for the home page.